uint64 timestamp 
int32 num
int8[] type
int8[] state
int8[] is_new
int8[] slot_id
int64 count
int8 target_id
int64 delay_time
geometry_msgs/PolygonStamped[] polygonStamped
